gsitemap.php

```diff
@@ -619,6 +620,7 @@ public function createSitemap($id_shop = 0)
         }
 
         $this->createIndexSitemap();
+        $this->indentXml();
```

Comment:
Isn't this already done by `formatXml()` here?

https://github.com/PrestaShop/gsitemap/blob/f65a769cdf0fbe25ba722ac0dc3584580d8711ee/gsitemap.php#L772

Reply:
For the first sitemap only.